modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -181,9 +186,9 @@ def prepare_for_upload(form)
                           form_id: get_form_id)
         location, uuid = lighthouse_service.request_upload
         stamp_pdf_with_uuid(form, uuid)
-        create_form_submission_attempt(uuid)
+        submission_attempt = create_form_submission_attempt(uuid)
```

Comment:
This looks like it is possibly a thing uploading to lighthouse, so probably not related to this backup code and only a change needed to support the backup code, but... if you are stamping the document with an upload ID are you generating a NEW pdf (with its own uuid stamped on it) for each upload, or generating 1 pdf, and attempting to upload multiple times (via a new lighthouse_service.request_upload), or are these not retried? 

Just making sure a PDF with 1 uuid stamped on it isnt getting uploaded to lighthouse to another uuid